cmd/openstack-cloud-controller-manager/main.go

```diff
@@ -101,21 +95,30 @@ the cloud specific control loops shipped with Kubernetes.`,
 
 			flag.PrintFlags(cmd.Flags())
 
-			c, err := s.Config(KnownControllers(), ControllersDisabledByDefault.List())
+			c, err := s.Config(controllerList, app.ControllersDisabledByDefault.List())
 			if err != nil {
 				fmt.Fprintf(os.Stderr, "%v\n", err)
 				os.Exit(1)
 			}
 
-			if err := app.Run(c.Complete(), wait.NeverStop); err != nil {
+			cloudconfig := c.Complete().ComponentConfig.KubeCloudShared.CloudProvider
+			cloud, err := cloudprovider.InitCloudProvider(cloudconfig.Name, cloudconfig.CloudConfigFile)
+			if err != nil {
+				fmt.Fprintf(os.Stderr, "%v\n", err)
+			}
+			if cloud == nil {
+				fmt.Fprintf(os.Stderr, "%v\n", err)
+			}
```

Comment:
I assume we should error out here and line 107?

Reply:
right. Ill update. Thanks